### PR TITLE
Knative operator

### PR DESCRIPTION
**Pull request by matzew**

### CODE REVIEW

File: ci-operator/config/openshift-knative/operator/openshift-knative-operator-release-0.14.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+base_images:
+  base:
+    cluster: https://api.ci.openshift.org
+    name: "4.3"
```
Comment (matzew) [inline ci-operator/config/openshift-knative/operator/openshift-knative-operator-release-0.14.yaml:4]: perhaps worth to go w/ 4.4 ?

File: ci-operator/config/openshift-knative/operator/openshift-knative-operator-release-0.14.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    namespace: ocp
+    tag: base
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.13
```
[the remark this answers is not in the document]
Comment (matzew) [reply ci-operator/config/openshift-knative/operator/openshift-knative-operator-release-0.14.yaml:12]: not yet

File: ci-operator/config/openshift-knative/operator/openshift-knative-operator-release-0.14.yaml (hunk 1 above)
Comment (markusthoemmes) [reply ci-operator/config/openshift-knative/operator/openshift-knative-operator-release-0.14.yaml:4]: To be decided I think. As long as we don't know where we'll support 1.8 it's not worth changing imo.

**Pull request merged**